src/lib/sendinblue.ts

```diff
@@ -13,90 +17,86 @@ const headers = {
   'api-key': token,
 }
 
-export interface EmailAttributes {
+export interface TemplateAttributes {
   firstname: string
   lastname: string
   newsletter: boolean
   deadline: string
   form: string
```

Comment:
If we have limited number of forms, it could be specified here: `form: 'postpone' | 'newsletter'`